src/Build.UnitTests/TerminalLoggerConfiguration_Tests.cs

```diff
@@ -232,6 +231,30 @@ public void TerminalLoggerDefaultOff(string defaultValue)
         ShouldNotBeTerminalLog(output);
     }
 
+    [WindowsFullFrameworkOnlyTheory]
+    [InlineData("1")]
+    [InlineData("0")]
+    public void TerminalLoggerOnInvalidProjectBuild(string msbuildinprocnodeState)
+    {
+        var projectFile = _env.CreateFile(_env.CreateFolder(createFolder: true), "myProjBroken.proj", $"""
+            <Project>
+              <Target Name='Build'>
+                <RegisterAssembly Assemblies="nonexistent.dll" />
+              </Target>
+            </Project>
```

Comment:
I suspect the repro project can be simplified to:

```xml
            <Project>
              <Target Name='Build' />
            </Project>
```

Reply:
It reports a successful build for this case.